db/migrate/20230406111020_add_can_sponsor_skilled_worker_visa_to_courses.rb

```diff
@@ -0,0 +1,5 @@
+class AddCanSponsorSkilledWorkerVisaToCourses < ActiveRecord::Migration[7.0]
+  def change
+    add_column :courses, :can_sponsor_skilled_worker_visa, :boolean
```

Comment:
Do these have a natural default value? If so they can be `null: false, default: false` to save dealing with `nil` values in application logic. If `nil` is a meaningful value, e.g. 'don't know' then its fine as is.

Reply:
i think it's probably legit to have `nil` values, eg for old courses that we have in apply but no longer sync from TTAPI